origin-discovery/src/listener/handler_marketplace.js

```diff
@@ -379,7 +383,7 @@ class NoGasMarketplaceEventHandler extends MarketplaceEventHandler {
       throw new Error(`ListingIpfs and Id mismatch: ${listingId} !== ${listing.creator.listing.createDate}`)
     }
 
-    if(listing.creator != offer.buyer) 
+    if(listing.creator != offer.buyer)
```

Comment:
nit: since we are changing this line, do you mind using the !== operator rather than != ?
same thing for line 388 below.

Reply:
I will do a prettier run after this PR is merged which should handle these.